src/tests/SharpRaven.UnitTests/SharpRaven.UnitTests.csproj

```diff
@@ -129,6 +129,9 @@
       <Name>SharpRaven</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
It is appeared after i installed NUnit Test Adapter (v2) and it seems is a Visual Studio behaviour https://github.com/nunit/nunit-vs-adapter/issues/51.